xskillscore/core/contingency.py

```diff
@@ -0,0 +1,787 @@
+from functools import wraps
+
+import numpy as np
+import xarray as xr
+
+from xhistogram.xarray import histogram as xhist
+
+__all__ = ['histogram', 'Contingency']
+
+OBSERVATIONS_NAME = 'observations'
+FORECASTS_NAME = 'forecasts'
+
+
+def histogram(*args, bins=None, bin_names=None, **kwargs):
```

Comment:
Should the user have access to this? The `__all__` hints at that, but it only seems like you'd use it internally through the `Contingency` class. If the latter is the case, I'd call it `_histogram`.